### PR TITLE
Implement WKUIDelegate methods to handle popups & prompts

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14544
affects: Student, Teacher
release note: Fixed certain links not opening in pages

### CODE REVIEW

File: Core/Core/Extensions/WKWebViewExtensions.swift [hunk 1]
```diff
@@ -28,6 +28,7 @@ extension WKWebView {
 
     public func handle(_ name: String, handler: @escaping MessageHandler) {
         let passer = MessagePasser(handler: handler)
+        configuration.userContentController.removeScriptMessageHandler(forName: name)
```
Comment (vanwagonet) [inline Core/Core/Extensions/WKWebViewExtensions.swift:31]: I got a crash when the new window tried to add handlers that (apparently) already existed.

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/50430977/p/2d0f55bb9fbd8af8e6687aced7902f1e">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/50430986/p/d98f60477e87c6c267ab6e5aed43eb68">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/50430955/p/29f81a67d2ca75a9fefbdc5ea29f0df4">
    
  </a>
</details>

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>



<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Core/Core/UIViews/CoreWebView.swift:340:5: Opening Brace Spacing Violation: Opening braces should be preceded by a single space and on the same line as the declaration. (opening_brace)
    [student]
    [teacher]
</details>


[MBL-14544](https://instructure.atlassian.net/browse/MBL-14544)

#### Affected Apps: Student, Teacher

#### Release Note: 
Fixed certain links not opening in pages
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by armstrongnate — APPROVED** [review]
Nice work!

**Pull request merged**